tests/generic/mongocursor-tailable-003.phpt

```diff
@@ -0,0 +1,115 @@
+--TEST--
+MongoCursor::tailable() with getNext() and hasNext() (initial query matches one document)
+--SKIPIF--
+<?php if (getenv('SKIP_SLOW_TESTS')) die('skip slow tests excluded by request'); ?>
+<?php require_once "tests/utils/standalone.inc" ?>
+--FILE--
+<?php
+require_once "tests/utils/server.inc";
+
+function log_getmore($server, $info) {
+    //echo "Issuing getmore\n";
+}
+
+$ctx = stream_context_create(array(
+    'mongodb' => array(
+        'log_getmore' => 'log_getmore',
+    ),
+));
+
+$host = MongoShellServer::getStandaloneInfo();
+$m = new MongoClient($host, array(), array('context' => $ctx));
+
+$db = $m->selectDB(dbname());
+$db->selectCollection(collname(__FILE__))->drop();
+$db->createCollection(collname(__FILE__), array('capped' => true, 'size' => 10240, 'max' => 3));
+
+$coll = $db->selectCollection(collname(__FILE__));
+$cursor = $coll->find()->tailable();
+
+$x = 0;
+$i = 0;
+
+$doInsert = function() use ($coll, &$x) {
+    $coll->insert(array('x' => $x));
+    printf("Inserted: x=%d\n", $x);
+    ++$x;
+};
+
+$doInsert();
```

Comment:
Per [the documentation](http://docs.mongodb.org/manual/tutorial/create-tailable-cursor/), a tailable cursor is immediately dead if the query has no initial match.